fly.toml

```diff
@@ -0,0 +1,22 @@
+# fly.toml app configuration file generated for knowledge-snowy-hill-4870 on 2024-06-22T12:26:58-04:00
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = 'knowledge-snowy-hill-4870'
```

Comment:
This file tells `fly` how to configure our app

We will also likely rebrand here, but didn't want to spend time on that while I was coding lol